---
title: "Fruit Data"
author: "Catherine Gubert"
date: "Friday, October 17, 2014"
output: html_document
---

When you click the **Knit** button a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. You can embed an R code chunk like this:

```{r}
summary(cars)
```

You can also embed plots, for example:

```{r, echo=FALSE}
plot(cars)
```

Note that the parameter was added to the code chunk to prevent printing of the R code that generated the plot.

##Measuring Fruit Size to Help Map Genetic Pathways in *Arabidopsis*

######Introduction
*Arabidopsis thaliana* has been a popular model organism for the study of genetic function due to its simplicity and knowledge of its genome. Cell separation and organ abscission is a common biological process among plants that allows them to shed organs such as ripened fruit or dead leaves. In an effort to understand a signaling pathway involved in this process, a set mutants known to affect this process were crossed together and the resulting phenotypes were analyzed. NEVERSHED (NEV) is thought to assist with trafficking of cargo between endosomes and the extracelluar space. Mutations in this gene block the shedding of outer floral organs in *Arabidopsis* flowers and reduce the length of mature fruit. EVERSHED (EVR) encodes a receptor-like kinase that, when supressed with *nev*, rescues abscission and organ shedding. However, it has also been observed that the *nev evr* double mutant does not rescue WT fruit length, suggesting that the pathways for control of abscission and fruit development are separate, though they are both affected by these two genes. Two other receptor-like kinases, HAESA (HAE) and HAESA-LIKE2(HSL2), act redundantly to switch on the production of enzymes required for cell separation. The peptide INFLORESCENCE DEFICIENT IN ABSCISSION (IDA) is the proposed ligand for both receptors. Loss of HAE/HSL2 function in *nev evr* flowers prevents organ shedding. To investigate the role of HAE and HSL2 in fruit development in relation to NEV, EVR, and IDA, the following mutant crosses were generated: *evr*, *nev*, *nev evr*, *nev evr hae*, *nev evr hae hsl2*, and *nev evr ida*. This report seeks to confirm that *nev* and *evr* significantly reduce fruit growth as compared to WT. It is expected that HAE and HSL2 act redundantly in controlling fruit growth. 
NULL HYP: 

Your document should layout your question in the introduction, provide a brief synopsis for why it is important, 

######Background and Materials
The raw data for this analysis were kindly shared by Dr. Sarah Liljegren at the University of Mississippi. All background information, methods, and references can be found in [HAESA and HAESA-LIKE2 activate organ abscission downstream of NEVERSHED and EVERSHED in *Arabidopsis* flowers](https://www.landesbioscience.com/journals/psb/article/29115/), a short communication published in Plant Signaling and Behavior, Landes Bioscience, 2014.

Using NIH ImageJ, `r nrow(fruitdata)` mature fruit of each genotype were measured lengthwise. The measurments of the mutant fruit were compared to the measurments of wild-type.

your general approach to analyzing the data,

######Statistical Methods

Loss of NEV or EVR activity reduces the length of fruit by `r percent[["nev"]]`% and `r percent[["evr"]]`%, repectively. 

```r
colMeans(fruitdata)
#plot fruitdata
barplot(means,axisnames=FALSE,col="dodgerblue3",ylim=c(0,14),main="Fruit Length",xlab="Genotype",ylab="length(mm)")
#create vector of bar mid-points
mp<-barplot(means,col="blue",ylim=c(0,14),main="Fruit Length",xlab="Genotype",ylab="length(mm)")
#label bars
axis(1,labels=c("WT","evr","nev","nev evr hae","nev evr hae hsl2","nev evr","nev evr ida"),font=3,at=mp)
#outline bar plot
box()
#get standard deviations of data
stDevs<-matrix(c(sd(fruitdata$WT),sd(fruitdata$evr),sd(fruitdata$nev),sd(fruitdata$nevevrhae),sd(fruitdata$nevevrhaehsl),sd(fruitdata$nevevr),sd(fruitdata$nevidaevr)))
#plot error bar segments
segments(mp,means-stDevs,mp,means+stDevs,lwd=1)
segments(mp-0.1,means-stDevs,mp+0.1,means-stDevs,lwd=1)
segments(mp-0.1,means+stDevs,mp+0.1,means+stDevs,lwd=1)
#compare means of mutant fruit length to WT
percent<-(means/(means["WT"]))*100
```

######Summary
a summary of your results, a discussion, and mention of what your future directions would be.
