---
title: "Fruit Data"
author: "Catherine Gubert"
date: "Friday, October 17, 2014"
output: html_document
---


##Measuring Fruit Size to Help Map Genetic Pathways in *Arabidopsis*

######Introduction

*Arabidopsis thaliana* has been a popular model organism for the study of genetic function due to its simplicity and knowledge of its genome. Cell separation and organ abscission is a common biological process among plants that allows them to shed organs such as ripened fruit or dead leaves. In an effort to understand a signaling pathway involved in this process, a set mutants known to affect this process were crossed together and the resulting phenotypes were analyzed. NEVERSHED (NEV) is thought to assist with trafficking of cargo between endosomes and the extracelluar space. Mutations in this gene block the shedding of outer floral organs in *Arabidopsis* flowers and reduce the length of mature fruit. EVERSHED (EVR) encodes a receptor-like kinase that, when supressed with *nev*, rescues abscission and organ shedding. However, it has also been observed that the *nev evr* double mutant does not rescue WT fruit length, suggesting that the pathways for control of abscission and fruit development are separate, though they are both affected by these two genes. Two other receptor-like kinases, HAESA (HAE) and HAESA-LIKE2(HSL2), act redundantly to switch on the production of enzymes required for cell separation. The peptide INFLORESCENCE DEFICIENT IN ABSCISSION (IDA) is the proposed ligand for both receptors. Loss of HAE/HSL2 function in *nev evr* flowers prevents organ shedding. To investigate the role of HAE and HSL2 in fruit development in relation to NEV, EVR, and IDA, the following mutant crosses were generated: *evr*, *nev*, *nev evr*, *nev evr hae*, *nev evr hae hsl2*, and *nev evr ida*. This report seeks to confirm that *nev* and *evr* significantly reduce fruit growth as compared to WT. It is expected that HAE and HSL2 act redundantly in controlling fruit growth, either to promote or reduce fruit length. 


######Background and Data

The raw data for this analysis were kindly shared by Dr. Sarah Liljegren at the University of Mississippi. All background information, methods, and references can be found in [HAESA and HAESA-LIKE2 activate organ abscission downstream of NEVERSHED and EVERSHED in *Arabidopsis* flowers](https://www.landesbioscience.com/journals/psb/article/29115/), a short communication published in Plant Signaling and Behavior, Landes Bioscience, 2014.

Using NIH ImageJ, `r nrow(fruitdata)` mature fruit of each genotype were measured lengthwise. The measurments of the mutant fruit were compared to the measurments of wild-type. Several theoretical maps of the signaling pathways involving NEV, EVR, HAE, HSL2, and IDA were deduced from organ shedding and fruit length data. Organ shedding data and proposed signaling pathways are not shown here. 


######Statistical Methods

```{r,echo=FALSE}
#reorder data frame
fruitdata<-fruitdata[c("WT","nev","evr","nevevr","nevevrhae","nevevrhaehsl","nevidaevr")]
means<-colMeans(fruitdata)
```

The original data set was separated into two groups, with the first containing data collected from WT, *evr*, *nev*, *nev evr hae*, and *nev evr hae hsl2* and the second group containing measurments from WT, *nev evr*, and *nev evr ida* fruit. For the purposes of this report, the measurments from the second set of WT fruit were not used. Although data from *nev evr* and *nev evr ida* fruit were collected on a separate date from the other data, all representative fruit was within the same growth stage. 

```{r} 
fruitdata
```

A bar plot was generated to illustrate the average lengths of mutant fruit as compared to wild-type. 

```{r,echo=FALSE}
#create barplot of fruit lengths
#create vector of bar mid-points to place error bars
mp<-barplot(means,names.arg=FALSE,col="dodgerblue3",ylim=c(0,15),main="Fruit Length",xlab="Genotype",ylab="length(mm)")
#label bars
axis(1,labels=c("WT","nev","evr","nev evr","nev evr hae","nev evr hae hsl2","nev evr ida"),at=mp)
#outline bar plot
box()
#get standard deviations of data
stDevs<-matrix(c(sd(fruitdata$WT),sd(fruitdata$evr),sd(fruitdata$nev),sd(fruitdata$nevevrhae),sd(fruitdata$nevevrhaehsl),sd(fruitdata$nevevr),sd(fruitdata$nevidaevr)))
rownames(stDevs)<-c("WT","nev","evr","nevevr","nevevrhae","nevevrhaehsl","nevidaevr")
colnames(stDevs)<-"SD"
#plot error bar segments
segments(mp,means-stDevs,mp,means+stDevs,lwd=1)
segments(mp-0.1,means-stDevs,mp+0.1,means-stDevs,lwd=1)
segments(mp-0.1,means+stDevs,mp+0.1,means+stDevs,lwd=1)
```

```{r, echo=FALSE}
#is difference in average lengths significant between genotypes?
#null hypothesis: the fruit lengths between genotypes are not significantly different
#use analysis of variance test
#get data into simple matrix to run ANOVA
groups<-c(rep("WT",length(fruitdata$WT)),rep("nev",length(fruitdata$nev)),rep("evr",length(fruitdata$evr)),rep("nevevr",length(fruitdata$nevevr)),rep("nevevrhae",length(fruitdata$nevevrhae)),rep("nevevrhaehsl",length(fruitdata$nevevrhaehsl)),rep("nevidaevr",length(fruitdata$nevidaevr)))
data<-c(fruitdata$WT,fruitdata$nev,fruitdata$evr,fruitdata$nevevr,fruitdata$nevevrhae,fruitdata$nevevrhaehsl,fruitdata$nevidaevr)
#run ANOVA
results<-anova(lm(data~groups))
```

An analysis of variance among all the data confirms that the difference in fruit length observed between the different genotypes is greater than the difference in fruit length among the individual samples from each genotype (F value: `r results[["F value"]]`)

```{r, echo=FALSE}
anova(lm(data~groups))
#F>1, so there is more difference between groups than within them
#reject null hypothesis
#pairwise t test will tell which ones are different from each other
```

Both NEV and EVR promote fruit growth, as indicated by reduction in fruit length in both mutants. Compared to WT, *nev* fruit are `r ((means["nev"])/(means["WT"])*100)`% as long (`r (means[["nev"]])` ± `r stDevs[2]`mm) and *evr* fruit are `r ((means["evr"])/(means["WT"])*100)`% as long (`r means[["evr"]]` ± `r stDevs[3]`mm). The reduction of growth in the double mutant is significantly greater than either of the individual mutants, supporting the hypothesis that NEV and EVR independently affect fruit growth. *nev evr* fruit are `r ((means["nevevr"])/(means["WT"])*100)`% as long as WT fruit (`r (means[["nev"]])` ± `r stDevs[4]`mm). Supporting previous research that shows HAE and HSL2 to function redundantly, the *nev evr hae hsl2* quadruple mutant shows a partial rescue of normal fruit length. *nev evr hae hsl2* fruit are `r ((means["nevevrhaehsl"])/(means["WT"])*100)`% of WT fruit length. The lengths of quadruple mutant fruits are more similar to the single mutants (`r ((means["nevevrhaehsl"])/(means["nev"])*100)`% of *nev* and `r ((means["nevevrhaehsl"])/(means["evr"])*100)`% of *evr*) than to the double mutant (`r ((means["nevevrhaehsl"])/(means["nevevr"])*100)`% of *nev evr*). Repression of IDA, the proposed ligand for HAE/HSL2 in the organ abscission pathway, partially restores fruit growth in *nev evr* plants. The triple mutants were `r ((means["nevidaevr"])/(means["nevevr"])*100)`% the length of the double mutants (p<0.05).

```{r,echo=FALSE}
t.test(fruitdata$nevidaevr,fruitdata$nevevr)
```

The *nev evr ida* mutant fruit are `r ((means["nevidaevr"])/(means["nevevrhaehsl"])*100)`% of *nev evr hae hsl2* fruit. The removal of IDA activity does not restore fruit growth as much as the blocking of HAE/HSL2. 

```{r,echo=FALSE}
t.test(fruitdata$nevevrhaehsl,fruitdata$nevidaevr)
```

A summary of statistical significance is shown by running a paired t test.

```{r,echo=FALSE}
pairwise.t.test(data,groups)
```



######Summary






